org-formation/710-tgw/_tasks.yaml

```diff
@@ -123,9 +126,12 @@ Mappings:
     VpcName:
       '055273631518': 'snowflakevpc'   # org-sagebase-scicomp
       '420786776710': 'bridge-aux'   # org-sagebase-bridgedev
-  TgwSpokesS:
+  TgwSpokesSynapseDev:
     VpcName:
       '449435941126': 'synapse-dev-vpc-2'      # org-sagebase-synapsedev
+  TgwSpokesSynapseProd:
+    VpcName:
+      '325565585839': 'synapse-prod-vpc-2'      # org-sagebase-synapseprod
```

Comment:
I see 'synapse-prod-vpc' (note: need to delete old VPC b4 merging if this is the case). Need to double-check in dev.

Reply:
Scratch, internal ref.